bin/utest/vm_map.c

```diff
@@ -0,0 +1,79 @@
+#include "utest.h"
+
+#include <assert.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+int test_sharing_memory_simple(void) {
+  char *map =
+    mmap(NULL, 4096, PROT_READ | PROT_WRITE, MAP_ANON | MAP_SHARED, 0, 0);
```

Comment:
> MAP_ANON
> Map anonymous memory not associated with any specific file. The file descriptor is not used for creating MAP_ANON regions, and must be specified as -1. The mapped memory will be zero filled.

mmap(2) - NetBSD Manual Pages

Should we stick to it?

Reply:
I changed it